core-rust/mesh-api-server/src/mesh_api/handlers/block_transaction.rs

```diff
@@ -6,7 +6,7 @@ pub(crate) async fn handle_block_transaction(
 ) -> Result<Json<models::BlockTransactionResponse>, ResponseError> {
     assert_matching_network(&request.network_identifier, &state.network)?;
 
-    let database = state.state_manager.database.snapshot();
+    let database = state.state_manager.database.access_direct();
```

Comment:
Ditto

Reply:
Decided to revert to `snapshot()` as per [this conversation](https://rdxworks.slack.com/archives/C071C6DB8TG/p1733841098517859?thread_ts=1732877188.486819&cid=C071C6DB8TG)